docs/src/constants/redirect-routes.js

```diff
@@ -81,10 +81,13 @@ export default {
   '/components/native/text': ROUTES.TEXT,
   '/components/native/text-input': ROUTES.TEXT_INPUT,
   '/components/native/text-view': ROUTES.TEXT_VIEW,
-  '/components/native/tappable-link-label': ROUTES.TAPPABLE_LINK_LABEL,
+  '/components/native/tappable-link-label': ROUTES.LINK,
   '/components/native/touchable-native-feedback':
     ROUTES.TOUCHABLE_NATIVE_FEEDBACK,
   '/components/native/touchable-overlay': ROUTES.TOUCHABLE_OVERLAY,
+  '/components/star-rating-interactive': ROUTES.STAR_RATING,
```

Comment:
These don't need to redirect as they are already using the correct path.

Reply:
This is a separate page so it would go to the interactive page and not rating as it was separate for Android 😄

Reply:
https://backpack.github.io/components/star-rating-interactive?platform=android

Reply:
https://backpack.github.io/components/star-rating?platform=android - interactive will now be located in this page correctly together now 😄

Reply:
Ok now I understand the point of this PR. I should have read the description 🤦